### DISCUSSION TITLE
how to delete partially downloaded files

### DISCUSSION THREAD

**Discussion by ghost** [aclap-dev/video-downloadhelper#1231] [Help and Q&A]
### Your question

i was downloading a file but my pc suddenly turned off. so the partially downloaded file is now sitting. is there any way to delete them. thanks. 

### Addon & CoApp version and details

version: 9.1.0.43
target: microsoft
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"D:\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36 Edg/128.0.0.0
{
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": true,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 4,
    "dable_attempts_last": 1725848806058
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Do you mean you can't delete the files in the Windows file manager?  Or do you mean there are leftovers in VDH?  For the operating system, I would suggest searching for an answer with a web search.  Not a search in this forum on GitHub.  A general web search.  If you're just talking about VDH, you can open the settings, click on `More settings` then scroll the settings all the way to the bottom.  There's a `Reload addon` button you should click.  That should clear out VDH.  I'm surprised this happened if you closed all browser windows then restarted Edge.
